pkg/graph/schema.resolvers.go

```diff
@@ -332,6 +332,18 @@ func (r *mutationResolver) CreateAccessibilityRequestDocument(ctx context.Contex
 		return nil, keyErr
 	}
 
+	accessibilityRequest, requestErr := r.store.FetchAccessibilityRequestByID(ctx, input.RequestID)
+	if requestErr != nil {
+		return nil, requestErr
+	}
+	ok, authErr := r.service.AuthorizeUserIs508TeamOrRequestOwner(ctx, accessibilityRequest)
```

Comment:
I would favor services.Authorize... over r.service.Authorize... if you can call the authorizer directly, I'd like to phase out having those services hang off the resolver

Reply:
Agreed! That's causing a test to fail, and I wanted to have solving that issue be part of some additional work. see https://github.com/CMSgov/easi-app/pull/1007